workflows/dragen-transcriptome-pipeline/4.0.3/dragen-transcriptome-pipeline__4.0.3.cwl

```diff
@@ -80,6 +97,11 @@ inputs:
     doc: |
       Mark identical alignments as duplicates
     type: boolean
+  enable_sort:
+    label: enable sort
+    doc: |
+      True by default, only set this to false if using --bam-input and --tumor-bam-input parameters
```

Comment:
Is --tumor-bam-input an option in Dragen WTS?

Reply:
No. Carried over from somatic pipeline. Updated in https://github.com/umccr/cwl-ica/pull/347/commits/723a29ef3e0254e6288592eb31dd0d0e639f0209